ChildMindInstitute/mindlogger-admin-OLD-Vue
# Add dashboard for token logger

- Added new route for the dashboard
- Added interactive token chart
- Added filtering by users and time range

Issue #207 

## Testing

1. Create an owner user `o1@test.com`
2. Create several users (e.g. `u1@test.com`, `u2@test.com`, `u3@test.com`)
3. With the owner account, import a TokenCollector applet, and invite the users from step 2 to the applet.
4. Create a reviewer user `r1@test.com`
5. Invite the reviewer user to the applet and grant him access to data from user 1 and 3 only.
6. Create a reviewer user `r2@test.com`
7. Invite the new reviewer user to the applet and grant him access to data from user 2 and 3.

The owner account should see the dashboard for all the users.
Reviewer 1 should see the dashboard for users 1 and 3.
Reviewer 2 should see the dashboard for users 2 and 3.
